include/deal.II/fe/fe_tools_extrapolate.templates.h

```diff
@@ -1152,6 +1152,10 @@ namespace FETools
       static Utilities::MPI::CollectiveMutex      mutex;
       Utilities::MPI::CollectiveMutex::ScopedLock lock(mutex, communicator);
 
+      // We pick a new tag in each round. Wrap around after 10 rounds:
+      const int mpi_tag =
+        Utilities::MPI::internal::Tags::fe_tools_extrapolate + round % 10;
```

Comment:
is ten enough?

Reply:
I have no idea how many rounds are needed or of wrapping around can be a problem, to be honest. What should we do?

Reply:
Does every round have some kind of synchronization point? Then this should be ok. I suspect that the ten rounds should synchronize enough, but who knows...

Reply:
I have no idea, I have to admit.